manual.md

```diff
@@ -49,13 +49,17 @@ RAD Governance Proposals (RGPs) must be formally labeled with the phase of the p
 
-During _Formal Review_, each proposal is required to publish a Snapshot poll to gauge consensus on whether it should proceed to an on-chain vote. This is the last chance for the community to propose changes and give feedback on the proposal before it moves into an on-chain vote.
+The *Discussion* phase is the primary time for the community to review and provide feedback on proposals. Proposal authors should try to include as much detail as possible in this first draft of their proposals. Proposal authors can use these templates to create their proposals, and should post them under [Proposal Discussions](https://community.radworks.org/c/governance/discussions/19) on the forum by the second Monday of each cycle (see 'Proposal Cycles' for details).  
 
-To initiate a Snapshot poll for a Formal Review, you must be added as an 'author' in our Snapshot space. This measure is in place to prevent spam proposals. Please reach out to Shelby (shelb_ee#9785 on Discord) with the Ethereum address of the party posting your proposal. All Snapshot Polls must begin on **5:00pm GMT+2** the **third Monday of the month** and end at **5:00pm GMT+2** on the **following Monday**. Each Snapshot poll must have three voting options: `Yes` `No` and `Abstain`.
+_Formal Review_ is the last chance for the community to propose changes and give feedback on the proposal before it moves to an on-chain vote. During _Formal Review_, proposal authors are required to publish an updated version of their proposal on the forum taking into account feedback from the *Discussion* phase. If applicable, the proposal must be accompanied by executable on-chain code for the community to review. See this template [INSERT LINK WHEN READY] for how the code should be presented. Proposals in this stage should be posted under the [Active Proposals](https://community.radworks.org/c/governance/proposals/18) category on the forum. 
+
+Each proposal is required to publish a Snapshot poll to gauge consensus on whether it should proceed to an on-chain vote. To initiate a Snapshot poll for a Formal Review, you must be added as an 'author' in our Snapshot space. This measure is in place to prevent spam proposals. Please reach out to Shelby (shelb_ee#9785 on Discord) with the Ethereum address of the party posting your proposal. All Snapshot Polls must begin on **5:00pm GMT+2** the **third Monday of the month** and end at **5:00pm GMT+2** on the **following Monday**. Each Snapshot poll must have three voting options: `Yes` `No` and `Abstain`. Proposal authors should include a link to the Snapshot poll at the top of their proposal on the forum. 
```

Comment:
This is a lot of text to have in a paragraph. Perhaps we list it as a set of steps instead?